src/azure-cli/azure/cli/command_modules/eventhubs/action.py

```diff
@@ -109,38 +109,71 @@ def __call__(self, parser, namespace, values, option_string=None):
+        from azure.cli.command_modules.eventhubs.constants import INCOMING_MESSAGES
+        from azure.cli.command_modules.eventhubs.constants import INCOMING_BYTES
+        from azure.cli.command_modules.eventhubs.constants import OUTGOING_BYTES
+        from azure.cli.command_modules.eventhubs.constants import OUTGOING_MESSAGES
+
+        throttling_policy = {
+            "name": None,
+            "type": "ThrottlingPolicy",
+            "throttling_policy": {
+                "rate_limit_threshold": None,
+                "metric_id": None
+            }
+        }
 
         for (k, v) in (x.split('=', 1) for x in values):
             if k == 'name':
-                name = v
+                throttling_policy["name"] = v
 
             elif k == 'rate-limit-threshold':
                 if v.isdigit() is False:
                     raise CLIError('rate-limit-threshold should be an integer')
-                rate_limit_threshold = int(v)
+                if 'throttling_policy' in throttling_policy:
+                    throttling_policy["throttling_policy"] = {}
+                throttling_policy["throttling_policy"]["rate_limit_threshold"] = int(v)
 
             elif k == 'metric-id':
                 if v.lower() == INCOMING_MESSAGES.lower():
-                    metric_id = INCOMING_MESSAGES
+                    throttling_policy["throttling_policy"]["metric_id"] = INCOMING_MESSAGES
                 elif v.lower() == INCOMING_BYTES.lower():
-                    metric_id = INCOMING_BYTES
+                    throttling_policy["throttling_policy"]["metric_id"] = INCOMING_BYTES
                 elif v.lower() == OUTGOING_MESSAGES.lower():
-                    metric_id = OUTGOING_MESSAGES
+                    throttling_policy["throttling_policy"]["metric_id"] = OUTGOING_MESSAGES
                 elif v.lower() == OUTGOING_BYTES.lower():
-                    metric_id = OUTGOING_BYTES
+                    throttling_policy["throttling_policy"]["metric_id"] = OUTGOING_BYTES
                 else:
                     raise CLIError('Only allowed values for metric_id are: {0}, {1}, {2}, {3}'.format(INCOMING_MESSAGES, INCOMING_BYTES, OUTGOING_MESSAGES, OUTGOING_BYTES))
 
             else:
                 raise InvalidArgumentValueError("Invalid Argument for:'{}' Only allowed arguments are 'name, rate-limit-threshold and metric-id'".format(option_string))
 
-        if (name is None) or (metric_id is None) or (rate_limit_threshold is None):
-            raise CLIError('One of the throttling policies is missing one of these parameters: name, metric-id, rate-limit-threshold')
+        if (throttling_policy["name"] is None) or (throttling_policy["throttling_policy"].get("rate_limit_threshold", None) is None) or (throttling_policy["throttling_policy"].get("metric_id", None) is None):
+            raise RequiredArgumentMissingError('Throttling policies is missing one of these parameters:name, metric-id, rate-limit-threshold')
```

Comment:
The linter error is raised from this line.

Reply:
Solved.